installer/templates/phx_assets/app.js

```diff
@@ -1,18 +1,22 @@
-// We need to import the CSS so that webpack will load it.
-// The MiniCssExtractPlugin is used to separate it out into
-// its own CSS file.
+// We import the CSS which is extracted to its own file by esbuild.
```

Comment:
```suggestion
// We import the CSS which is extracted to its own file by esbuild.
// Remove this line if you add a your own CSS build pipeline (e.g postcss)
```

I think this is a footgun if you end up also running postcss or tailwindcss and still have esbuild trying to bundle css also.